modules/openapi-generator/src/main/resources/typescript-node/models.mustache

```diff
@@ -68,6 +68,16 @@ let typeMap: {[index: string]: any} = {
     {{/models}}
 }
 
+// Check if a string starts with another string without using es6 features
+function startsWith(str: string, match: string): boolean {
+    return str.lastIndexOf(match, 0) === 0;
```

Comment:
That being said, is it really necessary to preserve pre-es6 compatibility here? `startsWith` and `endsWith` have been supported in Node.js and every major browser since 2015.